Codeinwp/neve
# [1/5][🎛 Customizer UI] Change all checkboxes to toggles

Part 1/5
---

### Summary
- chore: Removed some unused code (the Top_Bar class)
- chore: change all toggle controls to react based controls
- fix: about page links were linking to non-existent customizer sections


### Will affect visual aspect of the product

NO

### Test instructions


- Check that all theme defined checkboxes / old toggles are now react-based toggle controls.


Closes #1394.
